client/ayon_shotgrid/plugins/publish/collect_shotgrid_session.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import ayon_api
+from ayon_shotgrid.lib.credentials import get_local_login
```

Comment:
You should not use this in publish plugins. The function should be used only in AYON process with available `keyring` module, but publish plugins are used inside DCCs which is dangerous.

Reply:
I'll use the `OPENPYPE_SG_USER` env var which should be set by the login bit